src/java.base/share/classes/sun/security/provider/certpath/X509CertPath.java

```diff
@@ -26,7 +26,6 @@
 package sun.security.provider.certpath;
 
 import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
```

Comment:
The copyright year needs to be updated for this file and changed to 2021 in the other files where applicable.

Reply:
done